x-pack/plugins/cases/public/components/user_actions/description.test.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiCommentList } from '@elastic/eui';
+import { render, screen } from '@testing-library/react';
+
+import { Actions } from '../../../common/api';
+import { getUserAction } from '../../containers/mock';
+import { TestProviders } from '../../common/mock';
+import { createDescriptionUserActionBuilder } from './description';
+
+jest.mock('../../common/lib/kibana');
+jest.mock('../../common/navigation/hooks');
+
+describe('createDescriptionUserActionBuilder ', () => {
+  const handleOutlineComment = jest.fn();
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders correctly when editing a description', async () => {
+    const userAction = getUserAction('description', Actions.update);
+    // @ts-ignore no need to pass all the arguments
```

Comment:
I keep seeing this same `@ts-ignore` comment in many other tests and I am not a big fan of using them when we can avoid them. Could you perhaps create a common test helper builder that handles this and call it in these cases? that way we don't put so many `@ts-ignore` across the code.